classification/classifier.py

```diff
@@ -43,7 +45,10 @@ def classify(self):
     def is_page_stampifiable(self):
         ''' returns the is_stampifiable flag'''
         self.classify()
-        return self.is_stampifiable
+        if not self.is_stampifiable:
+            raise WebsiteNotStampifiableError(
+                message="Website cannot be stampified!",
+                failure_source="Summarizer")
```

Comment:
You can move this bottom of classify and remove `is_page_stampifiable(self)` function entirely.

Reply:
done!